src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/index.tsx

```diff
@@ -128,7 +121,12 @@ export const CreateNSTreeModal = ({
 
   const mutation = useCreateTree({
     componentOnError: () => {
-      setShouldShowErrorNotification(true);
+      dispatch(addNotification({
+        reduxId: Date.now(),
```

Comment:
That's the best convention I can think of for generating more or less unique ID (seems pretty unlikely two would be generated within the same millisecond, since notifs are always the result of user action and often involve travel over the network). I think it is important for the name of the prop to reflect what it represents. It's true that reduxId is a timestamp, but the point is that it uniquely identifies the notification.

Reply:
I can change it to notifId or something similar (unfortunately, the exposedprops that reduxnotification does not allow the use of `id` directly)